.gitignore

```diff
@@ -14,5 +14,8 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 
-# Logs
-*.log
\ No newline at end of file
+# The artifacts contains only logs, for now
+artifacts
+
+# OSX specific files
+.DS_Store
```

Comment:
As the person who doesn't know: What's that?

Reply:
At the risk of coming off brisk: I would recommend googling this :). These files pop up on my machine after certain command line operations.